plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1661,13 +1661,14 @@ private void prepareInsertExistingTable(HdfsContext context, TableAndMore tableA
             if (!targetPath.equals(currentPath)) {
                 asyncRename(hdfsEnvironment, renameExecutor, fileRenameCancelled, fileRenameFutures, context, currentPath, targetPath, tableAndMore.getFileNames().get());
             }
-            updateStatisticsOperations.add(new UpdateStatisticsOperation(
-                    tableAndMore.getIdentity(),
-                    table.getSchemaTableName(),
-                    Optional.empty(),
-                    tableAndMore.getStatisticsUpdate(),
-                    true));
-
+            if (!isAcidTransactionRunning()) {
```

Comment:
nit: tag with `// TODO(https://github.com/trinodb/trino/issues/7190) update table statistics for transactional tables`

Reply:
Tagged.